seed_stage_based_messaging/utils.py

```diff
@@ -55,4 +56,13 @@ def get_available_metrics():
         available_metrics.append(
             "subscriptions.message_set.{}.total.last".format(messageset_name))
 
+    subscriptions = Subscription.objects.all()
+    for subscription in subscriptions:
```

Comment:
These are systems where we can have 1mil+ subscriptions, so iterating through all of them will take very long and take a huge amount of memory. It will be better to let the database do it, something like: 

```
Subscriptions.objects.order_by('lang').distinct('lang').values_list('lang', flat=True)
```

Which should give you a list of languages, and do all of the removing duplicates on the database side.

Docs:
https://docs.djangoproject.com/en/1.10/ref/models/querysets/#distinct
https://docs.djangoproject.com/en/1.10/ref/models/querysets/#values-list